src/lamp_py/ingestion/glides.py

```diff
@@ -107,31 +107,28 @@ def append_records(self) -> None:
                 while start < now:
                     end = start + relativedelta(months=1)
                     if end < now:
-                        unique_table = (
```

Comment:
a lot of the same actions were taking place in either of the `if end < now` condition, so i collapsed them.